.prettierignore

```diff
@@ -97,3 +97,7 @@ tools/markdown-magic/test/include.md
 
 # These files are auto-generated according to the comments in the files
 **/charts/**/Chart.yaml
+
+# Prettier messes up the formatting and regular expressions of this file
+experimental/dds/tree2/.dependency-cruiser.js
+experimental/dds/tree2/.dependency-cruiser-known-violations.json
```

Comment:
Its surprising that this causes issues.

If there are issues with just a few lines, consider adding `// prettier-ignore` comments instead so the rest of the file can have auto-formatting.